src/table/components/virtualized-table/__tests__/Wrapper.spec.tsx

```diff
@@ -49,6 +58,8 @@ describe('<Wrapper />', () => {
         pagination: { borderColor: '' },
       },
     } as unknown as ExtendedTheme;
+
+    constraints = { active: false };
```

Comment:
Could be good to know: I think it is never false, it is either undefined or true